qemu/tests/x86_cpu_flags.py

```diff
@@ -22,6 +22,13 @@ def run(test, params, env):
     if check_host_flags:
         check_cpu_flags(params, flags, test)
 
+    unsupported_models = params.get("unsupported_models", "")
+    cpu_model = params.get("cpu_model")
+    if not cpu_model:
+        cpu_model = cpu.get_qemu_best_cpu_model(params)
+    if cpu_model in unsupported_models:
```

Comment:
`if cpu_model in unsupported_models.split()`

Reply:
update and thanks.
test PASS: (1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_smap: PASS (71.39 s)

Reply:
Pls update polarion test case as well, so they can be consistent.

Reply:
> Pls update polarion test case as well, so they can be consistent.

update and thanks.